test/functionalTest/cases/3122_location_metadata_issues/get_location_attr_with_V1_creation.test

```diff
@@ -0,0 +1,176 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This .test covers the following cases:

![imagen](https://user-images.githubusercontent.com/1534240/48832345-3ae96480-ed79-11e8-949b-7bead600651f.png)

Great!

NTC

Reply:
It's ok